packages/stage-ui/src/components/index.ts

```diff
@@ -1,8 +1,11 @@
+export * from './DataGui'
+export * from './FirstTimeSetup'
 export * from './Gadgets'
 export * from './Graphics'
 export * from './Layouts'
 export * from './Menu'
 export * from './Misc'
 export * from './Physics'
 export * from './Providers'
+export * from './Scenes'
```

Comment:
```suggestion
export * from './Providers'
```

Scenes requires Live2D and many ClientOnly renders which is not suitable for many external uses of `stage-ui`, especially not good for Histoire at this moment, let's keep it non-`export`ed.